test/integration/serverless/build_csharp_functions.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash
 
-echo "Building C# Lambda functions"
+echo "Building C# Lambda functions for $ARCHITECTURE architecture"
 cd src/csharp-tests
 dotnet restore --verbosity quiet
 set +e #set this so we don't exit if the tools are already installed
 dotnet tool install -g Amazon.Lambda.Tools --framework netcoreapp3.1 --verbosity quiet
 set -e
-dotnet lambda package --configuration Release --framework netcoreapp3.1 --verbosity quiet --output-package bin/Release/netcoreapp3.1/handler.zip
\ No newline at end of file
+
+if [ $ARCHITECTURE == "arm64" ]; then
+    dotnet lambda package --configuration Release --framework netcoreapp3.1 --verbosity quiet --output-package bin/Release/netcoreapp3.1/handler.zip --function-architecture arm64
+else
+    dotnet lambda package --configuration Release --framework netcoreapp3.1 --verbosity quiet --output-package bin/Release/netcoreapp3.1/handler.zip --function-architecture x86_64
+fi
```

Comment:
Nit: It looks like these lines are mostly the same, except for the one argument at the end. Maybe you can find a way to interpolate the architecture into the line without having the whole line written out twice?

Reply:
fixed